Lesson9/merve/styles.css

```diff
@@ -0,0 +1,172 @@
+* {
+    box-sizing: border-box;
+  }
+  body{
+  max-width: 100%;
+  display:flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  font-size: 1em;
+  font-family:  medium-content-sans-serif-font, -apple-system, BlinkMacSystemFont,    "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-   serif;
+  font-weight: 400;
+  letter-spacing: 0.01em;
+  line-height: 1.4em;
+  }
+  h1{
+  font-family: sohne, "Helvetica Neue", Helvetica, Arial, sans-serif;
+   font-weight: 800;
+   font-size: 42px;
+   line-height: 52px;
+   letter-spacing: -0.011em;
+   color: #242424;
+   margin-top: 1.19em;
+   margin-bottom: 32px;
+  }
+  .first-box{
+   max-width: 700px;
+  }
+  .first-img{
+  border-radius: 50px;
+  position: absolute;
```

Comment:
If you want to use position absolute for an element(s), you need to make sure that there's a container element that has position set to 'relative'. In this case, it's a div element with class 'social-media-profile'. 
```
.social-media-profile {
	position: relative;
}
.first-img {
	border-radius: 50px;
	position: absolute;
}
.second-img {
	position: absolute;
	border-radius: 50px;
	border: 3px solid rgb(255, 255, 255);
}

```

Then, adjust the position of the second-img using 'top' and 'left' values instead of using marign.